src/imu_vn_100.cpp

```diff
@@ -254,6 +254,10 @@ void ImuVn100::LoadParameters() {
   vpe_accel_adaptive_filtering_.c1 = declare_parameter("vpe.accel_tuning.adaptive_filtering.y", 4.0);
   vpe_accel_adaptive_filtering_.c2 = declare_parameter("vpe.accel_tuning.adaptive_filtering.z", 4.0);
 
+  hsi_mode_ = declare_parameter("hsi.mode", 1);
+  hsi_output_ = declare_parameter("hsi.output", 3);
+  hsi_converge_rate_ = declare_parameter("hsi.converge_rate", 5);
```

Comment:
One small comment here is that we should probably do error checking to make sure the passed in value is legal.  However, since we don't do that for the other declared parameters, I think we can just open an issue about it for now.

Reply:
Yes and yes.